rbd-target-api.py

```diff
@@ -876,6 +1029,11 @@ def _disk(image_id):
                 logger.error("LUN size unknown - {}".format(image_id))
                 return jsonify(message="LUN {} failure".format(mode)), 500
 
+            if 'owner' not in disk:
```

Comment:
Nit: when I attempted "/disks reconfigure <disk id> max_data_area_mb 128" against an image not yet assigned to a target, I hit: ```Failed to reconfigure : failed to deactivate disk: failed on ceph-iscsi0. LUN deactivate failure - Disk rbd.lun2 must be added to a target```

We might want a nicer error message like "cannot reconfigure until disk assigned to target". Also, now when creating disks, it probably doesn't make sense to allow any control overrides since they cannot be validated until the disk is mapped to a target in a later step.